### ISSUE TITLE
Semi-Visible Instances

### ISSUE THREAD

**Issue by brandleesee** [GeekJosh/BertUI#4]
OS: Windows 7
Setup: Laptop --> HDMI --> HDTV

![image](https://cloud.githubusercontent.com/assets/12674437/7898263/f6f7e886-06fa-11e5-9489-fe144b636321.png)

BertUI Instances: 1
CSV Lines of Code: 7
Icons Visible: 3/7

Textual Explanation: Top 4 icons are not completely visible. Have re-downloaded .zip and inserted .csv, same result.

End: I understand that the current download at your main site will soon be foresaken, however it is optimum for my intended usage. Any help is appreciated.

**Comment by brandleesee** [comment]
Update 1: The same issue happens if CSV Lines of Code are 10 or 11 whereas if 9 or 12, the software functions correctly except that the top edges are omitted (see attached pic.)

![image](https://cloud.githubusercontent.com/assets/12674437/7900963/8327e216-0773-11e5-9c5a-2df63c5d4a33.png)

**Comment by GeekJosh** [comment]
Could you please tell me what your screen's resolution is? Also, do you have any DPI scaling enabled?

The first issue certainly looks like a bug, whereas the second is a limitation at the moment - there is a minimum size the buttons will scale to, so too many buttons simply get cropped.

Will look in to solving both issues as soon as I have chance.

**Comment by brandleesee** [comment]
Laptop screen resolution is: 1366 x 768
HDTV: 39 inches

As for DPI, at the moment it is set to the default: 100%. The testing was done on 100%.
However, I may increase it to better accommodate the minute desktop fonts when using the HDTV as the main display. If you recommend to keep DPI at 100%, I'll stick to your advice and look for another solution relating to larger but neat fonts.

**Comment by GeekJosh** [comment]
Thanks for the extra info. Theoretically, DPI shouldn't have any effect BertUI, but I've done zero testing (hence the question)

If you do find more issues with non-default DPI settings, please do report them. XAML is supposed to handle scaling gracefully but I would not be surprised if it doesn't

**Comment by brandleesee** [comment]
As soon as I experiment with increased DPI, I'll post my experience on this thread.

While testing I tried using some autohotkey scripts to stretch BertUI to use the whole screen in hopes of restoring the cropped buttons. That simply did not happen. I now understand that the software implements a ratio/percentage relative to the screen's resolution.

I also tried another script to give the window the Minimize, Maximize and Close buttons. That also did not happen. Its necessity was to relieve myself from right-clicking on the taskbar where BertUI is so as to close it for re-testing an amendment.

The last two paragraphs are just so you know I tried to cheat a way towards more comfort without waiting for a software update!
